Integrations/Gmail/Gmail.py

```diff
@@ -424,14 +424,19 @@ def users_to_entry(title, response):
 
         })
     headers = ['Type', 'ID', 'Username',
-               'DisplayName', 'Groups', 'CustomerId', 'Domain', 'OrganizationUnit', 'Email', 'VisibleInDirectory']
+               'DisplayName', 'Groups', 'CustomerId', 'Domain', 'OrganizationUnit', 'Email', 'VisibleInDirectory',
+               'nextPageToken']
```

Comment:
`nextPageToken` shouldn't be in the table entry

Reply:
will change